c7n/resources/s3.py

```diff
@@ -1162,15 +1162,32 @@ class ToggleLogging(BucketActionBase):
 
     def process(self, resources):
         enabled = self.data.get('enabled', True)
-
+        # code to support expand_variables()
+        session = local_session(self.manager.session_factory)
+        iam_client = session.client('iam')
+        aliases = iam_client.list_account_aliases().get('AccountAliases', ('',))
```

Comment:
do you need the account aliases? their mutable, so can cause unexpected problems when log locations change. general practice from aws around logs seems to primarily utilize account id.

Reply:
OH please please build this feature into the master code!  Having to manage my own s3 function code for just logging sucks.  Having it built in makes my life much easier! 👍

Reply:
@jtroberts83 are you referencing using account aliases, or just account ids?

Reply:
Client wanted account aliases as their S3 Logging standard already used them, as people could remember where to go based on account alias, while always having to remember the account id was seen as slowing people down.   Since this code is specific to a single action, and I doubt many deployments run this action in a time critical way, can we leave AsIs?  It's only "slower" for when a change needs to be made, which is unlikely to occur in a quantity large enough to cause problems.

Reply:
noted and fair enough, just wanted to ensure it was being used. the issues with aliases, is  per previous comment, their mutable, so very easy to 'lose' logs, on some org change for display labels on accounts.

Reply:
one thing we do need then is to extend action's permissions to include the iam.GetAccountAliases permission